.github/ISSUE_TEMPLATE/1.bug.yml

```diff
@@ -34,17 +34,19 @@ body:
     attributes:
       label: Steps to reproduce 🕹
       description: |
-        Provide a link to a live example (you can use codesandbox.io) and an unambiguous set of steps to reproduce this bug.
-        Include code to reproduce, if relevant (which it most likely is).
+        Please provide a link to a live example and an unambiguous set of steps to reproduce this bug.
+             
+        As a starting point, we recommend you browse our [documentation](https://mui.com/x/advanced-components/), and [select](https://mui.com/static/x/forking-an-example.png) the closest example to your use case.
 
-        You can use the official codesandbox template of the [DataGrid](https://codesandbox.io/s/datagrid-v5-quick-start-54iz1) or [DataGridPro](https://codesandbox.io/s/datagridpro-v5-quick-start-5pjhc) as a starting point.
+        Or you can use a [basic template](https://codesandbox.io/s/mui-x-basic-template-uh3pm2?file=/src/App.tsx) to build a reproduction case.
```

Comment:
Looking at this demo, I wonder if it shouldn't go in the opposite direction compared to the codesandbox edit buttons. The alternative would be to keep it an open pristine playground. Taking https://mui.com/r/issue-template and adding all the dependencies that developer might need, joy, system, base, data grid, date picker, tree view, etc.

Reply:
Yeah, I was wondering what would work better for the devs on this case. I considered it'd be faster to have a couple of commented lines to support a "ready-to-start" template with different components. On MUI X this is still viable but with MUI Core also in mind, it's definitely not a good start ^^.

Reply:
>  I considered it'd be faster to have a couple of commented lines to support a "ready-to-start" template with different components

My flow of thought was: the codesandbox leads to the same outcome as the first proposed solution, so it's redundant so better remove it. But then, I was almost like, well, it's also a pain to have to open a codesandbox just to remove all the content on it, so I can copy & paste code from my own codebase.